GETTING_STARTED.md

```diff
@@ -0,0 +1,30 @@
+The simplest way to start to use 'AppKid' is an example app like:
+
+https://github.com/smumriak/AppKidDemo
+
+...which you can [after installing dependances] simply git-clone to your local machine, and 'swift run' it.  
+
+git clone https://github.com/smumriak/AppKidDemo 
+cd AppKidDemo
+swift build 
+swift run
```

Comment:
Code blocks should be distinguished by everything else by using code block markdown syntax that looks like this:
\`\`\`
your code
\`\`\`

Reply:
done